doc/components/popup.md

```diff
@@ -0,0 +1,481 @@
+# Popup
+
+- Spawns a buffer inside a popup
+- You can use `vim.api` to manipulate it like any other buffer
+
+## Properties & Usage
```

Comment:
Let's call these "Options" (like it's called in the codebase) instead of "Properties".